packages/web-frontend/cypress/scripts/generateConfig/generateOverlayConfig.js

```diff
@@ -28,19 +60,43 @@ const validateUrlGenerationOptions = options => {
   });
 };
 
-const generateUrlsUsingOptions = async (db, options) => {
+const objectifyUrl = url => {
```

Comment:
Some unhappy connotation with `objectification` here... But it sounds cool in code (?)

Reply:
That's just what they thought when implementing master and slave servers, and look how that's ended up 😝

If is a nice parallel to `stringifyUrl`... `JSON` library uses `JSON.parse(string)` and `JSON.stringify(object)`... how does `parseUrl(string)` sound?

Reply:
Sounds good! Can change that